src/VirtoCommerce.CustomerExportImportModule.Core/Models/CsvMember.cs

```diff
@@ -84,6 +84,13 @@ public abstract class CsvMember : IExportable, IHasOuterId, IHasDynamicPropertie
         [Ignore, JsonIgnore]
         public string ObjectType { get; set; }
 
+        [Ignore, JsonIgnore]
+        public bool AddressIsNotEmpty => Array.Exists(new[]
```

Comment:
Need to convert this property to a method. 
Need to rename it so it doesn't contain `Not`, becasue `if (!AddressIsNotEmpty)` looks weird.
Also, it would be better to avoid creating a new array every time.